### ISSUE TITLE
Confusing/nonsensical explanation of partial derivative of loss?

### ISSUE THREAD

**Issue by roaneg** [VikParuchuri/zero_to_gpt#14]
Lesson 2, talking about gradient descent, says:

> Because you add `b` to `x * w` to get the prediction, any change to `b` directly impacts the prediction.  So the partial derivative of `b` with respect to the loss is the same as the loss!

I'm a beginner with no formal experience in machine learning, though I do have some experience with calculus. The above quote talks about partial derivative of `b` with respect to loss, but the previous text talks about partial derivative of loss with respect to `b`. And in both the case of partial of loss with respect to `b` and partial of `b` with respect to loss, the resulting partial derivative is not equal to the loss function.

Hope I'm not making a naive beginner mistake with this understanding. Would be happy to be educated here. I've asked AI to try to educate me and it also thinks this section doesn't quite make sense, for whatever that's worth.
